src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2536,6 +2536,12 @@ limitations under the License.
         <deviceId editable="false">0x007B</deviceId>
         <class>Simple</class>
         <scope>Endpoint</scope>
+        <endpointComposition>
+            <compositionType>tree</compositionType>
+            <endpoint conformance="M" constraint="min 1">
```

Comment:
Followup: The "max 1 Cooktop" bit needs to be here too, right?

Reply:
Yes, it should be there. Fixing.

Reply:
Fixed, thank you.